tests/utils/normalizeclipboarddata.js

```diff
@@ -17,4 +17,44 @@ describe( 'normalizeClipboardData', () => {
 			normalizeClipboardData( '<span class="Apple-converted-space"> </span>x<span class="Apple-converted-space">\u00a0</span>' )
 		).to.equal( ' x ' );
 	} );
+
+	it( 'should strip all spans with no attributes', () => {
```

Comment:
And as for features which want to produce empty spans with no classes/attributes (cause that's what cleaned here), they will simply be incorrect. Just like you mentioned – they'd be treated as a garbage.